src/wallet/api/wallet2_api.h

```diff
@@ -67,6 +70,54 @@ enum NetworkType : uint8_t {
         bool set;
     };
 
+/**
+* brief: EnoteDetails - Container for all the necessary information that belongs to an enote
+*/
+struct EnoteDetails
+{
+    // QUESTION : If this struct is done, should I create the files "src/wallet/api/enote_details.[h/cpp]", put all the member variables inside the header and only set virtual getter functions in here, like it's done for other structs below?
```

Comment:
Difficult question. I would tend to keep the "style" of using getters, to have things consistent, even if we probably agree that the whole programming pattern is overkill.

Maybe something to bring up in the next meeting?